core/src/main/resources/error/error-classes.json

```diff
@@ -1051,6 +1051,16 @@
           "expects a binary value with 16, 24 or 32 bytes, but got <actualLength> bytes."
         ]
       },
+      "AES_IV_LENGTH" : {
+        "message" : [
+          "supports 16-byte CBC IVs and 12-byte GCM IVs, but got <actualLength> bytes for <mode>."
+        ]
+      },
+      "AES_SALTED_MAGIC" : {
```

Comment:
Is it used somewhere? If not, please, remove it.

Reply:
I think this was re-added by mistake when I merged this on top of SPARK-43286. My mistake.